src/golang/cmd/server/handler/get_workflow_dag_result.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/google/uuid"
```

Comment:
handler comment fix

Reply:
The naming of these routes is a little confusing to me.

I take it that the new idea here is for this to return a particular DagResult from a workflow given that the client already knows the id of the DagResult? Which is particularly useful for our details page of course.

How do we go about getting the list of Dags (executed, or unexecuted) when all we have is a workflow's id?

Reply:
This is another route (get_workflow) which we don't have time to refactor yet, and that route should give all dag results that stores to the same redux store as this handler.

This handler will still be useful for example, the user directly open the detail page from a link, like in a notification